cypress/plugins/bdd.ts

```diff
@@ -1,12 +1,35 @@
 const browserify = require('@cypress/browserify-preprocessor');
 const cucumber = require('cypress-cucumber-preprocessor').default;
 const path = require('path');
+const axios = require('axios');
+
 
 module.exports = (on, config) => {
+
   const options = {
     ...browserify.defaultOptions,
     typescript: path.join(path.resolve('..'), 'kiali-ui/node_modules/typescript'),
   };
 
   on('file:preprocessor', cucumber(options));
+
+  async function exportConfig() {
+    const getAuthStrategy = async (url: string) => {
+      try {
+        const resp = await axios.get(url+'/api/auth/info')
+        return resp.data.strategy
+      } catch (err) {
+        console.error(`ERROR: Kiali API is not reachable at ${JSON.stringify(err.config.url)}`);
+        throw new Error(`Kiali API is not reachable at ${JSON.stringify(err.config.url)}`)
+      }
+    }
+
+    config.baseUrl = config.env.BASEURL || config.baseUrl
```

Comment:
This isn't a blocker but I don't see why you need `config.env.BASEURL` when setting the env var `CYPRESS_BASE_URL` will populate `config.baseUrl`.